lib/internal/errors.js

```diff
@@ -1104,18 +1104,28 @@ E('ERR_INVALID_PACKAGE_CONFIG', (path, message, hasMessage = true) => {
 }, Error);
 E('ERR_INVALID_PACKAGE_TARGET',
   (pkgPath, key, subpath, target, base = undefined) => {
+    const relError = typeof target === 'string'
+      && target.length && !target.startsWith('./');
     if (key === null) {
       if (subpath !== '') {
         return `Invalid "exports" target ${JSONStringify(target)} defined ` +
         `for '${subpath}' in the package config ${pkgPath} imported from ` +
-        base;
+        `${base}.${relError ? ' - targets must start with "./"' : ''}`;
       } else {
         return `Invalid "exports" main target ${target} defined in the ` +
-          `package config ${pkgPath} imported from ${base}.`;
+          `package config ${pkgPath} imported from ${base}.${relError ?
+          ' - targets must start with "./"' : ''}`;
       }
     } else if (key === '.') {
       return `Invalid "exports" main target ${JSONStringify(target)} defined ` +
-      `in the package config ${pkgPath}${sep}package.json`;
+      `in the package config ${pkgPath}${sep}package.json${relError ?
+            ' - targets must start with "./"' : ''}`;
+    } else if (typeof target === 'string' && target !== '' &&
+      !target.startsWith('./')) {
+        return `Invalid "exports" target ${JSONStringify(target)} defined for '${
```

Comment:
There seems to be an inconsistency with the quote style being used. Is this supposed to be single quoted, double quoted, or unquoted?

Reply:
technically only curly quotes are correct in prose, never `'` or `"` :-)

Reply:
similar to below, this was not something introduced in this PR and I think if we want to fix this we should do it in a follow up

Reply:
Would it be worthwhile to open an issue about it first? There doesn't seem to be consistency amongst the other errors either. Technically it's not a breaking change, however, folks who parse their error logs may be interested in prior notice.

Reply:
A single PR to get quoting / formatting consistent between modules errors sounds like a good idea to me.

Reply:
- [x] I would like to work on this issue and submit a pull request.

I'd be making the PR once this one is successful. Keep me posted on whether it seems likely anyone would be available for review. The goal would be to match the convetion seen in the following image by circumventing the non-ascii-character lint rule via Unicode escape sequences.

![image](https://user-images.githubusercontent.com/17770407/79641274-76ea4a00-8164-11ea-9c3f-5ff6361f4247.png)

Reply:
- [x] I would like to work on this issue and submit a pull request.

I'd be making the PR once this one is successful. Keep me posted on whether it seems likely anyone would be available for review. The goal would be to match the convention seen in the following image by circumventing the `non-ascii-character` lint rule via Unicode escape sequences.

![image](https://user-images.githubusercontent.com/17770407/79641543-2247ce80-8166-11ea-841d-9fd31a046cf3.png)